README.md

```diff
@@ -224,6 +224,8 @@ errors. The [@sentry/rollup-plugin](#) is used to upload source maps.
 - Searching for schemas in the Topics and Schemas views is limited to the `subject` field only.
   Searching by other fields, such as `id` and `version`, is not supported due to cost and
   performance considerations.
+- The Message Viewer does not support consuming records that were compressed with `snappy`
```

Comment:
perhaps a note on if we plan to support it in the future?

Reply:
Good point! I'll add a link to the issue in the sidecar repository.